src/main/java/io/anserini/qa/passage/ScoredPassage.java

```diff
@@ -0,0 +1,30 @@
+package io.anserini.qa.passage;
+
+public class ScoredPassage implements Comparable<ScoredPassage> {
+  String sentence;
+  double score;
+
+  public ScoredPassage(String sentence, double score) {
+    this.sentence = sentence;
+    this.score = score;
+  }
+
+  public String getSentence() {
+    return  sentence;
+  }
+
+  public double getScore() {
+    return  score;
+  }
```

Comment:
@lintool do you want the methods to be written single line? This convention is consistent with the one followed throughout Anserini.